modules/setup/config/__init__.py

```diff
@@ -58,6 +61,7 @@ async def create(args):
         exchange_str = config["exchange"]
         config_module.timeframe = config["timeframe"]
         config_module.timeframe_ms = parse_timeframe(config_module.timeframe)
+        ConfigModule.strategy_name = config['strategy-name']
```

Comment:
```suggestion
        config_module.strategy_name = config['strategy-name']
```

Shouldn't it modify the instance of the class, not the main class itself?

Reply:
Changes have been made.